src/run.js

```diff
@@ -498,6 +501,10 @@ const minimalcss = async options => {
     // Here's the crucial part. Decide whether to keep the selector
     // Find at least 1 DOM that contains an object that matches
     // this selector string.
+    if (isInWhiteList(selectorString)) {
```

Comment:
The inclusion of these 4 lines now make it so that the code comment above (up to line 503) seems to belong to these 4 lines. Only. Can you re-arrange that a bit?